visualization/FairnessDashboard/src/Localization/zh-CN.json

```diff
@@ -0,0 +1,154 @@
+{
+  "loremIpsum": "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.",
+  "defaultClassNames": "类 {0}",
+  "defaultFeatureNames": "敏感特征 {0}",
+  "defaultSingleFeatureName": "敏感特征",
+  "defaultCustomMetricName": "自定义指标 {0}",
+  "accuracyTab": "性能的公平性",
+  "opportunityTab": "机会的公平性",
+  "modelComparisonTab": "模型比较",
+  "tableTab": "详细信息视图",
+  "dataSpecifications": "数据统计信息",
+  "attributes": "特性",
+  "singleAttributeCount": "1 个敏感特征",
+  "attributesCount": "{0} 个敏感特征",
+  "instanceCount": "{0} 个实例",
+  "close": "关闭",
+  "calculating": "正在计算...",
+  "accuracyMetric": "性能指标",
+  "errorOnInputs": "输入出错。目前敏感特征必须是分类值。请将值映射到已装箱的类别，然后重试。",
+  "Accuracy": {
+    "header": "你想如何衡量性能?",
+    "modelMakes": "模型",
+    "modelsMake": "模型",
+    "body": "你的数据包含 {0} 个标签和 {2} {1} 预测。根据此信息，我们推荐以下指标。请从列表中选择一个指标。",
+    "binaryClassifier": "二元分类器",
+    "probabalisticRegressor": "Probit 回归量",
```

Comment:
nit: probabilistic - but if it's hard to fix the name then leave it for a future PR